plugins/eksporter.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: eksporter
+spec:
+  version: "v1.0.0"
+  platforms:
+  - selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    uri: https://github.com/Kyrremann/kubeflow-eksporter/releases/download/v1.0.0/eksporter.tar.gz
+    sha256: "1c13fac59c1c306db8498f394ee4cf1ba98eaa44c6f404fc2fad505533f00835"
+    files:
+    - from: "/eksporter.rb"
+      to: "."
+    bin: "./eksporter.rb"
+  shortDescription: Eksport resources without generated fields
+  homepage: https://github.com/Kyrremann/kubeflow-eksporter
+  caveats: |
+    This plugin needs the following programs:
+    * ruby
+  description: |
+    This plugin helps you export resources, and trims away the fields
+    not necessary for applying the resource back into the cluster.
```

Comment:
So from the looks of your code it seems you are trying to solve the problem, where a resource is exported from one cluster/namespace and re-applied in another namespace. Is that correct? If so, you should explicitly mention this use-case here and in the `shortDescription`